wiki/es/es-Installation-for-Linux.md

```diff
@@ -98,12 +98,12 @@ Si tienes problemas con cortes de sonido (en particular errores de XRUN mostrado
 
 ## ¿Todo instalado?
 Mírate la [Configuración de Hardware](Hardware-Setup).
-Para información detallada sobre la utilización de Jamulus, por favor consulta el [Manual de Ayuda](https://github.com/corrados/jamulus/blob/master/src/res/homepage/manual.md).
+Para información detallada sobre la utilización de Jamulus, por favor consulta el [Manual de Ayuda](Software-Manual).
```

Comment:
Not sure about that yet. We might change this in future #110 but at the moment, it should stay like this.

Reply:
Done. Also in other documents where I'd changed it.

Reply:
Ok. I hope we will be able to restructure the page and rethink it. I'm currently working on another problem: We're a bit out of sync with the changes branch #128